commit f520116e9a532d057a389ad78729da4b1845bd32 (refs/changes/43/343/3)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2016-08-24T21:13:25+00:00 (3 years, 1 month ago)
    
    MORAY-354 Moray should ship client tools
    Reviewed by: Dave Pacheco <dap@joyent.com>
    Approved by: Dave Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 85cd0e2..5dd5033 100644
--- a/package.json
+++ b/package.json
@@ -20,6 +20,7 @@
         "lru-cache": "2.5.0",
         "node-manatee": "git+https://github.com/joyent/node-manatee.git#152d3e1",
         "manatee": "git+https://github.com/joyent/manatee.git#8615736",
+        "moray": "git+https://github.com/joyent/node-moray.git#fd5781b",
         "microtime": "0.5.1",
         "once": "1.3.0",
         "panic": "0.2.1",
